### PR TITLE
adding RF69 sketch and header files and folder

### PR DESCRIPTION
**Pull request by JBreithaupt**
RadioControl_rf69.h contains shared constants and data structure definitions for the RF69 receiver and transmitter sketches. Constants include pin assignments for inputs and outputs, encryption key, radio frequency, and UART baudrate.

RadioControl_rf69/RadioControl_rf69.ino contains fictions for the receiver and transmitter SAMD21 arduinos. The transmitter sketch is designed to transmit a struct of user inputs to the receiver unit at a frequency set by the BACKOFF constant. If the transmitter receives a reply from the receiver, it turns on a light to show the connection is up. The receiver sketch constantly checks for packets from the transmitter and sends a reply when a packet is received. Data are stored and accessed on both ends using structs for receiver variables and transmitter variables.

### CODE REVIEW

**Comment by ptressel** [comment]
Ok, I see what you mean now.  I was just looking at the fact that there were two commits and both after the last commit in the main repo.  I didn't notice that one of them is a merge commit...  So, you're right -- let's see if we can back out the merge commit.  It doesn't appear necessary, as there were no conflicts with your file, or at least, that's what the note from GitHub says...  But we should check.  Could you have a look at the list of files in the merge commit, and see if you had changes in any?  To see the list, click on the merge commit link, then under the "this merge commit was added cleanly..." message, click the "view the diff" link.  Let me know if any of those files are yours, and we'll figure out what to do with the merge commit.

**Comment by JBreithaupt** [comment]
Pat: please take another look, I've now cherry-picked the changes per instructions

File: RadioControl_rf69/RadioControl_rf69.ino [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  }
+
+  if (txData.autonomous) {
+    SerialUSB.print("A,");
+  }
+  else {
+    SerialUSB.print(",");
+  }
+  if (newline)
+    SerialUSB.print("\n");
```
Comment (ptressel) [inline RadioControl_rf69/RadioControl_rf69.ino]: Just FYI, if you use println() that will use whatever the system's newline is.  Also, if you could have prints in interrupt routines, then those can sneak in between your separate print(...) calls.  You could avoid that by forming the whole string first, and then doing one print(...).  E.g. put "E" or "" in a variable, and "A" or "" in another, in your if statements, then line up the whole schmeer as a string expression.

**Comment by ptressel** [comment]
Looks a lot tidier now -- no merge commit!  Thanks!

**Comment by ptressel** [comment]
Now...don't hit me, but...  Tyler snuck three commits into the main repo.  Could you try a normal rebase to pull those in?  Thanks!

**Pull request merged**